src/tests/BuildWasmApps/Wasm.Build.Tests/BlazorWasmBuildPublishTests.cs

```diff
@@ -284,6 +284,9 @@ private string CreateProjectWithNativeReference(string id)
         {
             CreateBlazorWasmTemplateProject(id);
 
+            string extraProps = @"
+                <EmccExtraLDFlags>-sERROR_ON_UNDEFINED_SYMBOLS=0</EmccExtraLDFlags>
```

Comment:
Is this still needed?

Reply:
I think you tested it recently and it was still needed, or was it the other test?

Reply:
Nice, we can leave just the other workaround and remove this one.